automated-comments/go/two-fer/find_minimal_conditional.md

```diff
@@ -0,0 +1,5 @@
+Unless it makes the code more readable, repetition should be avoided.
+Try to find the _minimal conditional_ to make the code DRY (Don't Repeat Yourself).
+
+Note: DRY isn't always better. It may sometimes be better to duplicate code snippets rather than to introduce 
```

Comment:
All of this is absolutely correct and wise advice, but maybe this is a little too much nuance for absolute beginners on the first exercise. (More or less any amount of nuance is usually too much in that situation.)

If you say to an experienced Gopher "Sometimes it's good to repeat yourself, and other times it isn't, depending on the situation," she will nod wisely and know exactly what you mean. But saying that to a beginner just leaves them mystified.

I'm not sure exactly what triggers this comment, so I can't necessarily suggest a better phrasing. But it might be something like "Short and simple code is more readable. Try to find the _minimal conditional_ here which passes the test."

Reply:
I agree. This is a targeted message to `two-fer`.

Expect the student to know programming, and thus be at least somewhat familiar with DRY and YAGNI. Perhaps:

```md
It is unnecessary to duplicate code in this instance. Find the _minimal condition_ to make the 
code DRY (Don't Repeat Yourself).
```

Reply:
I also agree that while DRY indeed isn't always better, perhaps we should not at this point bother the student with this subtle nuance?

Reply:
Agree that we should keep this short, concise, and specific to the two-fer case. 👍